File: Python/High Scores.py
def latest(scores):
    return scores[-1]

def personal_best(scores):
    scores = sorted(scores)
    return scores[-1]

def personal_top_three(scores, top_three = None):
    top_three = []
    scores = sorted(scores)
    if len(scores) < 3:
        top = len(scores)
    else:
        top = 3
    for i in range(-1, -top-1, -1):
        top_three.append(scores[i])
    return top_three

File: C#/High Scores.cs
using System;
using System.Collections.Generic;
using System.Linq;

public class HighScores
{
    public List<int> scoreList;

    public HighScores(List<int> list)
    {
        this.scoreList = list;
    }

    public List<int> Scores() => scoreList;

    public int Latest() => scoreList[scoreList.Count - 1];

    public int PersonalBest() => scoreList.Max();

    public List<int> PersonalTopThree() => scoreList.OrderByDescending(x => x).Take(3).ToList();
}